### PR TITLE
Add new test for java, hello-world

### LINKED ISSUE #597: [test] java-hello-world
Create a simple hello-world application to ensure that the language can compile (if necessary) and run (see ./test/tests/hylang-sh for an example of using expected-std-out.txt).

### PR DESCRIPTION
**Pull request by yosifkit**
uimanager test is from https://github.com/docker-library/openjdk/issues/129. Also hello-world test too! Fixes #597

Run with current official images. (still need to run on older images to see if they have the same errors)  And then fix the errors.
```console
$ ./test/run.sh -t java-hello-world -t java-uimanager-font $(bashbrew list openjdk --uniq --apply-constraints 2>/dev/null)
testing openjdk:6b38-jdk
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...passed
testing openjdk:6b38-jre
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...passed
testing openjdk:7u131-jdk
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...passed
testing openjdk:7u131-jdk-alpine
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...Could not get the default font's family
java.lang.UnsatisfiedLinkError: /usr/lib/jvm/java-1.7-openjdk/jre/lib/amd64/libfontmanager.so: Error relocating /usr/lib/jvm/java-1.7-openjdk/jre/lib/amd64/libfontmanager.so: AWTFontDefaultChar: symbol not found
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary1(ClassLoader.java:1968)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1893)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1875)
	at java.lang.Runtime.loadLibrary0(Runtime.java:849)
	at java.lang.System.loadLibrary(System.java:1088)
	at sun.font.FontManagerNativeLibrary$1.run(FontManagerNativeLibrary.java:61)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerNativeLibrary.<clinit>(FontManagerNativeLibrary.java:32)
	at sun.font.SunFontManager$1.run(SunFontManager.java:339)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.SunFontManager.<clinit>(SunFontManager.java:335)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:278)
	at sun.font.FontManagerFactory$1.run(FontManagerFactory.java:82)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.awt.Font.getFont2D(Font.java:490)
	at java.awt.Font.getFamily(Font.java:1219)
	at java.awt.Font.getFamily_NoClientCode(Font.java:1193)
	at java.awt.Font.getFamily(Font.java:1185)
	at container.main(container.java:21)
Error making a Metal look/feel, setting it to the UIManager, or getting its font...
java.lang.NoClassDefFoundError: Could not initialize class sun.awt.X11FontManager
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:278)
	at sun.font.FontManagerFactory$1.run(FontManagerFactory.java:82)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.awt.Font.getFont2D(Font.java:490)
	at java.awt.Font.getFamily(Font.java:1219)
	at java.awt.Font.getFamily_NoClientCode(Font.java:1193)
	at java.awt.Font.getFamily(Font.java:1185)
	at container.main(container.java:35)
failed
testing openjdk:7u131-jre
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...passed
testing openjdk:7u131-jre-alpine
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...Could not get the default font's family
java.lang.UnsatisfiedLinkError: /usr/lib/jvm/java-1.7-openjdk/jre/lib/amd64/libfontmanager.so: Error relocating /usr/lib/jvm/java-1.7-openjdk/jre/lib/amd64/libfontmanager.so: AWTFontDefaultChar: symbol not found
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary1(ClassLoader.java:1968)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1893)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1875)
	at java.lang.Runtime.loadLibrary0(Runtime.java:849)
	at java.lang.System.loadLibrary(System.java:1088)
	at sun.font.FontManagerNativeLibrary$1.run(FontManagerNativeLibrary.java:61)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerNativeLibrary.<clinit>(FontManagerNativeLibrary.java:32)
	at sun.font.SunFontManager$1.run(SunFontManager.java:339)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.SunFontManager.<clinit>(SunFontManager.java:335)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:278)
	at sun.font.FontManagerFactory$1.run(FontManagerFactory.java:82)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.awt.Font.getFont2D(Font.java:490)
	at java.awt.Font.getFamily(Font.java:1219)
	at java.awt.Font.getFamily_NoClientCode(Font.java:1193)
	at java.awt.Font.getFamily(Font.java:1185)
	at container.main(container.java:21)
Error making a Metal look/feel, setting it to the UIManager, or getting its font...
java.lang.NoClassDefFoundError: Could not initialize class sun.awt.X11FontManager
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:278)
	at sun.font.FontManagerFactory$1.run(FontManagerFactory.java:82)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.awt.Font.getFont2D(Font.java:490)
	at java.awt.Font.getFamily(Font.java:1219)
	at java.awt.Font.getFamily_NoClientCode(Font.java:1193)
	at java.awt.Font.getFamily(Font.java:1185)
	at container.main(container.java:35)
failed
testing openjdk:8u131-jdk
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...passed
testing openjdk:8u131-jdk-alpine
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...Could not get the default font's family
java.lang.UnsatisfiedLinkError: /usr/lib/jvm/java-1.8-openjdk/jre/lib/amd64/libfontmanager.so: Error relocating /usr/lib/jvm/java-1.8-openjdk/jre/lib/amd64/libfontmanager.so: AWTFontDefaultChar: symbol not found
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1941)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1845)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at sun.font.FontManagerNativeLibrary$1.run(FontManagerNativeLibrary.java:61)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerNativeLibrary.<clinit>(FontManagerNativeLibrary.java:32)
	at sun.font.SunFontManager$1.run(SunFontManager.java:339)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.SunFontManager.<clinit>(SunFontManager.java:335)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at sun.font.FontManagerFactory$1.run(FontManagerFactory.java:82)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.awt.Font.getFont2D(Font.java:491)
	at java.awt.Font.getFamily(Font.java:1220)
	at java.awt.Font.getFamily_NoClientCode(Font.java:1194)
	at java.awt.Font.getFamily(Font.java:1186)
	at container.main(container.java:21)
Error making a Metal look/feel, setting it to the UIManager, or getting its font...
java.lang.NoClassDefFoundError: Could not initialize class sun.awt.X11FontManager
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at sun.font.FontManagerFactory$1.run(FontManagerFactory.java:82)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.awt.Font.getFont2D(Font.java:491)
	at java.awt.Font.getFamily(Font.java:1220)
	at java.awt.Font.getFamily_NoClientCode(Font.java:1194)
	at java.awt.Font.getFamily(Font.java:1186)
	at container.main(container.java:35)
failed
testing openjdk:8u131-jre
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...Could not get the default font's family
java.awt.AWTError: Assistive Technology not found: org.GNOME.Accessibility.AtkWrapper
	at java.awt.Toolkit.loadAssistiveTechnologies(Toolkit.java:807)
	at java.awt.Toolkit.getDefaultToolkit(Toolkit.java:886)
	at sun.swing.SwingUtilities2.getSystemMnemonicKeyMask(SwingUtilities2.java:2020)
	at javax.swing.plaf.basic.BasicLookAndFeel.initComponentDefaults(BasicLookAndFeel.java:1158)
	at javax.swing.plaf.metal.MetalLookAndFeel.initComponentDefaults(MetalLookAndFeel.java:431)
	at javax.swing.plaf.basic.BasicLookAndFeel.getDefaults(BasicLookAndFeel.java:148)
	at javax.swing.plaf.metal.MetalLookAndFeel.getDefaults(MetalLookAndFeel.java:1577)
	at javax.swing.UIManager.setLookAndFeel(UIManager.java:539)
	at javax.swing.UIManager.setLookAndFeel(UIManager.java:579)
	at javax.swing.UIManager.initializeDefaultLAF(UIManager.java:1349)
	at javax.swing.UIManager.initialize(UIManager.java:1459)
	at javax.swing.UIManager.maybeInitialize(UIManager.java:1426)
	at javax.swing.UIManager.getDefaults(UIManager.java:659)
	at javax.swing.UIManager.getFont(UIManager.java:672)
	at container.main(container.java:21)
failed
testing openjdk:8u131-jre-alpine
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...Could not get the default font's family
java.lang.UnsatisfiedLinkError: /usr/lib/jvm/java-1.8-openjdk/jre/lib/amd64/libfontmanager.so: Error relocating /usr/lib/jvm/java-1.8-openjdk/jre/lib/amd64/libfontmanager.so: AWTFontDefaultChar: symbol not found
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1941)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1845)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at sun.font.FontManagerNativeLibrary$1.run(FontManagerNativeLibrary.java:61)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerNativeLibrary.<clinit>(FontManagerNativeLibrary.java:32)
	at sun.font.SunFontManager$1.run(SunFontManager.java:339)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.SunFontManager.<clinit>(SunFontManager.java:335)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at sun.font.FontManagerFactory$1.run(FontManagerFactory.java:82)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.awt.Font.getFont2D(Font.java:491)
	at java.awt.Font.getFamily(Font.java:1220)
	at java.awt.Font.getFamily_NoClientCode(Font.java:1194)
	at java.awt.Font.getFamily(Font.java:1186)
	at container.main(container.java:21)
Error making a Metal look/feel, setting it to the UIManager, or getting its font...
java.lang.NoClassDefFoundError: Could not initialize class sun.awt.X11FontManager
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at sun.font.FontManagerFactory$1.run(FontManagerFactory.java:82)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.awt.Font.getFont2D(Font.java:491)
	at java.awt.Font.getFamily(Font.java:1220)
	at java.awt.Font.getFamily_NoClientCode(Font.java:1194)
	at java.awt.Font.getFamily(Font.java:1186)
	at container.main(container.java:35)
failed
testing openjdk:9-b177-jdk
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...Could not get the default font's family
java.lang.InternalError: java.lang.reflect.InvocationTargetException
	at java.desktop/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:86)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.desktop/sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.desktop/java.awt.Font.getFont2D(Font.java:496)
	at java.desktop/java.awt.Font.getFamily(Font.java:1409)
	at java.desktop/java.awt.Font.getFamily_NoClientCode(Font.java:1383)
	at java.desktop/java.awt.Font.getFamily(Font.java:1375)
	at container.main(container.java:21)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:488)
	at java.desktop/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:84)
	... 7 more
Caused by: java.lang.NullPointerException
	at java.desktop/sun.awt.FontConfiguration.getVersion(FontConfiguration.java:1288)
	at java.desktop/sun.awt.FontConfiguration.readFontConfigFile(FontConfiguration.java:225)
	at java.desktop/sun.awt.FontConfiguration.init(FontConfiguration.java:107)
	at java.desktop/sun.awt.X11FontManager.createFontConfiguration(X11FontManager.java:765)
	at java.desktop/sun.font.SunFontManager$2.run(SunFontManager.java:440)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.desktop/sun.font.SunFontManager.<init>(SunFontManager.java:385)
	at java.desktop/sun.awt.FcFontManager.<init>(FcFontManager.java:35)
	at java.desktop/sun.awt.X11FontManager.<init>(X11FontManager.java:56)
	... 12 more
Error making a Metal look/feel, setting it to the UIManager, or getting its font...
java.lang.InternalError: java.lang.reflect.InvocationTargetException
	at java.desktop/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:86)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.desktop/sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.desktop/java.awt.Font.getFont2D(Font.java:496)
	at java.desktop/java.awt.Font.getFamily(Font.java:1409)
	at java.desktop/java.awt.Font.getFamily_NoClientCode(Font.java:1383)
	at java.desktop/java.awt.Font.getFamily(Font.java:1375)
	at container.main(container.java:35)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:488)
	at java.desktop/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:84)
	... 7 more
Caused by: java.lang.NullPointerException
	at java.desktop/sun.awt.FontConfiguration.getVersion(FontConfiguration.java:1288)
	at java.desktop/sun.awt.FontConfiguration.readFontConfigFile(FontConfiguration.java:225)
	at java.desktop/sun.awt.FontConfiguration.init(FontConfiguration.java:107)
	at java.desktop/sun.awt.X11FontManager.createFontConfiguration(X11FontManager.java:765)
	at java.desktop/sun.font.SunFontManager$2.run(SunFontManager.java:440)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.desktop/sun.font.SunFontManager.<init>(SunFontManager.java:385)
	at java.desktop/sun.awt.FcFontManager.<init>(FcFontManager.java:35)
	at java.desktop/sun.awt.X11FontManager.<init>(X11FontManager.java:56)
	... 12 more
failed
testing openjdk:9-b177-jre
	'java-hello-world' [1/2]...passed
	'java-uimanager-font' [2/2]...Could not get the default font's family
java.lang.InternalError: java.lang.reflect.InvocationTargetException
	at java.desktop/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:86)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.desktop/sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.desktop/java.awt.Font.getFont2D(Font.java:496)
	at java.desktop/java.awt.Font.getFamily(Font.java:1409)
	at java.desktop/java.awt.Font.getFamily_NoClientCode(Font.java:1383)
	at java.desktop/java.awt.Font.getFamily(Font.java:1375)
	at container.main(container.java:21)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:488)
	at java.desktop/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:84)
	... 7 more
Caused by: java.lang.NullPointerException
	at java.desktop/sun.awt.FontConfiguration.getVersion(FontConfiguration.java:1288)
	at java.desktop/sun.awt.FontConfiguration.readFontConfigFile(FontConfiguration.java:225)
	at java.desktop/sun.awt.FontConfiguration.init(FontConfiguration.java:107)
	at java.desktop/sun.awt.X11FontManager.createFontConfiguration(X11FontManager.java:765)
	at java.desktop/sun.font.SunFontManager$2.run(SunFontManager.java:440)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.desktop/sun.font.SunFontManager.<init>(SunFontManager.java:385)
	at java.desktop/sun.awt.FcFontManager.<init>(FcFontManager.java:35)
	at java.desktop/sun.awt.X11FontManager.<init>(X11FontManager.java:56)
	... 12 more
Error making a Metal look/feel, setting it to the UIManager, or getting its font...
java.lang.InternalError: java.lang.reflect.InvocationTargetException
	at java.desktop/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:86)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.desktop/sun.font.FontManagerFactory.getInstance(FontManagerFactory.java:74)
	at java.desktop/java.awt.Font.getFont2D(Font.java:496)
	at java.desktop/java.awt.Font.getFamily(Font.java:1409)
	at java.desktop/java.awt.Font.getFamily_NoClientCode(Font.java:1383)
	at java.desktop/java.awt.Font.getFamily(Font.java:1375)
	at container.main(container.java:35)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:488)
	at java.desktop/sun.font.FontManagerFactory$1.run(FontManagerFactory.java:84)
	... 7 more
Caused by: java.lang.NullPointerException
	at java.desktop/sun.awt.FontConfiguration.getVersion(FontConfiguration.java:1288)
	at java.desktop/sun.awt.FontConfiguration.readFontConfigFile(FontConfiguration.java:225)
	at java.desktop/sun.awt.FontConfiguration.init(FontConfiguration.java:107)
	at java.desktop/sun.awt.X11FontManager.createFontConfiguration(X11FontManager.java:765)
	at java.desktop/sun.font.SunFontManager$2.run(SunFontManager.java:440)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.desktop/sun.font.SunFontManager.<init>(SunFontManager.java:385)
	at java.desktop/sun.awt.FcFontManager.<init>(FcFontManager.java:35)
	at java.desktop/sun.awt.X11FontManager.<init>(X11FontManager.java:56)
	... 12 more
failed
```

### CODE REVIEW

**Comment by tianon** [comment]
I'm still a bit dubious of this `java-uimanager-font` bug -- IMO it's a feature that we've got one less package installed by default, so maybe this test should be instead installing that package somehow and verifying that it _then_ works as expected?

File: test/tests/run-in-container.sh [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
 # NOT INTENDED TO BE USED AS A TEST "run.sh" DIRECTLY
 # SEE OTHER "run-*-in-container.sh" SCRIPTS FOR USAGE
 
+# arguments to docker
+args=()
+opts="$(getopt -o '+' --long 'docker-arg:' -- "$@")"
+eval set -- "$opts"
+
+while true; do
+	flag=$1
```
Comment (tianon) [inline test/tests/run-in-container.sh]: `$1` should be quoted here: `flag="$1"`

File: test/tests/run-in-container.sh [hunk 2]
```diff
@@ -1,9 +1,30 @@
...
+	shift
+	case "$flag" in
+		--docker-arg) args+=( $1 ) && shift ;;
```
Comment (tianon) [inline test/tests/run-in-container.sh]: `$1` should be quoted here: `args+=( "$1" )`

**Comment by yosifkit** [comment]
Done, just hello world now

**Comment by tianon** [comment]
LGTM :+1:

Build test of #3201; e54917dd (`openjdk`):

```console

$ bashbrew build openjdk:6b38-jdk
Using bashbrew/cache:a24088fd0044b017abae114c8e810d43bbb61a3f66e06ab8b9f671669f61d443 (openjdk:6b38-jdk)
Tagging openjdk:6b38-jdk
Tagging openjdk:6b38
Tagging openjdk:6-jdk
Tagging openjdk:6

$ test/run.sh openjdk:6b38-jdk
testing openjdk:6b38-jdk
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed


$ bashbrew build openjdk:6b38-jre
Using bashbrew/cache:bdc352083afa58651697949b349423b24af7c42798edfe3e74270eaa5b417b5f (openjdk:6b38-jre)
Tagging openjdk:6b38-jre
Tagging openjdk:6-jre

$ test/run.sh openjdk:6b38-jre
testing openjdk:6b38-jre
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed


$ bashbrew build openjdk:7u131-jdk
Using bashbrew/cache:c403ec3eff3583bf1c933c1948c3720e3dc085fdfa1cce27e37979d86e860a68 (openjdk:7u131-jdk)
Tagging openjdk:7u131-jdk
Tagging openjdk:7u131
Tagging openjdk:7-jdk
Tagging openjdk:7

$ test/run.sh openjdk:7u131-jdk
testing openjdk:7u131-jdk
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed


$ bashbrew build openjdk:7u131-jdk-alpine
Using bashbrew/cache:9071d2d1e53a51344027eb4bd5fa817be06a0f5065f5aa7b258a800aa828dc52 (openjdk:7u131-jdk-alpine)
Tagging openjdk:7u131-jdk-alpine
Tagging openjdk:7u131-alpine
Tagging openjdk:7-jdk-alpine
Tagging openjdk:7-alpine

$ test/run.sh openjdk:7u131-jdk-alpine
testing openjdk:7u131-jdk-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed


$ bashbrew build openjdk:7u131-jre
Using bashbrew/cache:5029acdb415610a646eb19c0d3b0568862d866ab18b6ce234b86814aff252aaf (openjdk:7u131-jre)
Tagging openjdk:7u131-jre
Tagging openjdk:7-jre

$ test/run.sh openjdk:7u131-jre
testing openjdk:7u131-jre
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed


$ bashbrew build openjdk:7u131-jre-alpine
Using bashbrew/cache:2f6d559bb7a8523abb1799afd8da293822b7f052b9229c9abbebdcd27aa183fe (openjdk:7u131-jre-alpine)
Tagging openjdk:7u131-jre-alpine
Tagging openjdk:7-jre-alpine

$ test/run.sh openjdk:7u131-jre-alpine
testing openjdk:7u131-jre-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed


$ bashbrew build openjdk:8u131-jdk
Using bashbrew/cache:e3995f05f2190de3e6162616edcf86e48cb7a575fb4240a26e90de34918b93eb (openjdk:8u131-jdk)
Tagging openjdk:8u131-jdk
Tagging openjdk:8u131
Tagging openjdk:8-jdk
Tagging openjdk:8
Tagging openjdk:jdk
Tagging openjdk:latest

$ test/run.sh openjdk:8u131-jdk
testing openjdk:8u131-jdk
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed


$ bashbrew build openjdk:8u131-jdk-alpine
Using bashbrew/cache:75f4a7b036de006b027ba48ceffde167a626a24c239abd87c7379372203c8545 (openjdk:8u131-jdk-alpine)
Tagging openjdk:8u131-jdk-alpine
Tagging openjdk:8u131-alpine
Tagging openjdk:8-jdk-alpine
Tagging openjdk:8-alpine
Tagging openjdk:jdk-alpine
Tagging openjdk:alpine

$ test/run.sh openjdk:8u131-jdk-alpine
testing openjdk:8u131-jdk-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed


$ bashbrew build openjdk:8u131-jdk-windowsservercore
skipping "openjdk:8u131-jdk-windowsservercore" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh openjdk:8u131-jdk-windowsservercore
testing openjdk:8u131-jdk-windowsservercore
	image does not exist!


$ bashbrew build openjdk:8u131-jdk-nanoserver
skipping "openjdk:8u131-jdk-nanoserver" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh openjdk:8u131-jdk-nanoserver
testing openjdk:8u131-jdk-nanoserver
	image does not exist!


$ bashbrew build openjdk:8u131-jre
Using bashbrew/cache:dbe6068e0eac903aa59ed34d0bee31e6ebfc0de67131667cb4f92829b5dce98f (openjdk:8u131-jre)
Tagging openjdk:8u131-jre
Tagging openjdk:8-jre
Tagging openjdk:jre

$ test/run.sh openjdk:8u131-jre
testing openjdk:8u131-jre
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed


$ bashbrew build openjdk:8u131-jre-alpine
Using bashbrew/cache:5d8b921b67ad23d8b27f260882f6955f14c77510ffaa0c8668ceb0e0a5bf1abe (openjdk:8u131-jre-alpine)
Tagging openjdk:8u131-jre-alpine
Tagging openjdk:8-jre-alpine
Tagging openjdk:jre-alpine

$ test/run.sh openjdk:8u131-jre-alpine
testing openjdk:8u131-jre-alpine
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed


$ bashbrew build openjdk:9-b177-jdk
Using bashbrew/cache:1eeaaa8e71612c0a7e75173891bf16cb86bd0e3c0406a03acc4c33daf4c0906c (openjdk:9-b177-jdk)
Tagging openjdk:9-b177-jdk
Tagging openjdk:9-b177
Tagging openjdk:9-jdk
Tagging openjdk:9

$ test/run.sh openjdk:9-b177-jdk
testing openjdk:9-b177-jdk
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed


$ bashbrew build openjdk:9-b154-jdk-windowsservercore
skipping "openjdk:9-b154-jdk-windowsservercore" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh openjdk:9-b154-jdk-windowsservercore
testing openjdk:9-b154-jdk-windowsservercore
	image does not exist!


$ bashbrew build openjdk:9-b154-jdk-nanoserver
skipping "openjdk:9-b154-jdk-nanoserver" (due to architecture "amd64"; only "windows-amd64" supported)

$ test/run.sh openjdk:9-b154-jdk-nanoserver
testing openjdk:9-b154-jdk-nanoserver
	image does not exist!


$ bashbrew build openjdk:9-b177-jre
Using bashbrew/cache:98f51c22fc8aff9f045d0251bd5ffde326066e515d04cbe260cfe8d5e1495b33 (openjdk:9-b177-jre)
Tagging openjdk:9-b177-jre
Tagging openjdk:9-jre

$ test/run.sh openjdk:9-b177-jre
testing openjdk:9-b177-jre
	'utc' [1/5]...passed
	'cve-2014--shellshock' [2/5]...passed
	'no-hard-coded-passwords' [3/5]...passed
	'override-cmd' [4/5]...passed
	'java-hello-world' [5/5]...passed

```


The following images failed at least one test:

- `openjdk`:
  - `openjdk:8u131-jdk-windowsservercore`
  - `openjdk:8u131-jdk-nanoserver`
  - `openjdk:9-b154-jdk-windowsservercore`
  - `openjdk:9-b154-jdk-nanoserver`

**Pull request merged**